### PR TITLE
Core 11.6.0

### LINKED ISSUE #8020: Apparently wrong sysctl.d optimization settings
### Bug description

kernel.pid_max cannot be set to a value greater than PAGE_SIZE * 8 (usually 32768) in a Linux kernel with CONFIG_BASE_SMALL=1 enabled. So systemd will complain about invalid argument with larger values.
On 64-bit architectures, the default value of kernel.pid_max is 4194304, which is quite large and definitely enough. Limiting the value doesn't help either.
In other cases, the default value of kernel.pid_max is 32768, so there may be some room for optimization on 32-bit architectures.

Enabling net.ipv4.tcp_tw_recycle is harmful and the option is removed from Linux 4.12.

net.ipv6.conf.xxx.rp_filter has never appeared in the Linux kernel. It was never implemented. However there are replacements in ip6tables/nft.

### Steps to reproduce

Not relevant

### Update channel(s)

stable

### Package and version

```bash
aosc-aaa 11.5.2
should affect Retro/Afterglow too.
```

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

- aosc-aaa: update to 11.6.0
- glibc: add recommends for glibc-legacy
    Optional dependency for legacy compatibility.
- glibc-legacy: new
    This package provides legacy compatibility symlinks for removed glibc sonames.
- isl: update to 0.27
- aosc-aaa: change sysctl parameters
    - Remove inappropriate kernel.pid_max, net.ipv4.tcp_tw_recycle parameters.
    - Remove nonexistent net.ipv6.conf.all.rp_filter parameter.
    Ref: https://github.com/AOSC-Dev/aosc-os-abbs/issues/8020

Package(s) Affected
-------------------

- aosc-aaa: 11.6.0
- glibc: 1:2.38-7
- glibc-legacy: 0
- isl: 1:0.27

Security Update?
----------------

No

Build Order
-----------

```
#buildit isl glibc-legacy glibc aosc-aaa
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Review by eatradish — DISMISSED** [review]

**Comment by MingcongBai** [comment]
Tested good on LoongArch, JetBrains IntelliJ IDEA launches out-of-the-box (previously requiring `libutil.so`):

Screenshot_20240920_170438

Also, no more errors from `systemd-sysctl.service`.

File: core-libs/glibc-legacy/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+abinfo "Creating compatibility symlinks for glibc runtime libraries ..."
+mkdir -pv "$PKGDIR"/usr/lib/legacy/glibc
+for i in libanl.so.1 libdl.so.2 librt.so.1 libutil.so.1 libnsl.so.1; do
+    abinfo "Creating compatibility symlinks: $i ..."
+    ln -sv ../libc.so.6 \
```
Comment (KexyBiscuit) [inline core-libs/glibc-legacy/autobuild/build]:
```suggestion
    ln -sv ../../libc.so.6 \
```

**Review by KexyBiscuit — CHANGES_REQUESTED** [review]
```shell
kexybiscuit@ProArtB550-CREATOR [ ~ ] $ cd /usr/lib/legacy/glibc/
kexybiscuit@ProArtB550-CREATOR [ glibc ] $ ls -al
total 8
drwxr-xr-x 2 root root 4096 Sep 20 17:10 .
drwxr-xr-x 3 root root 4096 Sep 20 17:10 ..
lrwxrwxrwx 1 root root   12 Sep 19 22:33 libanl.so.1 -> ../libc.so.6
lrwxrwxrwx 1 root root   12 Sep 19 22:33 libdl.so.2 -> ../libc.so.6
lrwxrwxrwx 1 root root   12 Sep 19 22:33 libnsl.so.1 -> ../libc.so.6
lrwxrwxrwx 1 root root   12 Sep 19 22:33 librt.so.1 -> ../libc.so.6
lrwxrwxrwx 1 root root   12 Sep 19 22:33 libutil.so.1 -> ../libc.so.6
kexybiscuit@ProArtB550-CREATOR [ glibc ] $ file libanl.so.1
libanl.so.1: broken symbolic link to ../libc.so.6
```

**Review by KexyBiscuit — APPROVED** [review]
LGTM.

```shell
kexybiscuit@ProArtB550-CREATOR [ ~ ] $ file /usr/lib/legacy/glibc/libutil.so.1
/usr/lib/legacy/glibc/libutil.so.1: symbolic link to ../../libc.so.6
kexybiscuit@ProArtB550-CREATOR [ ~ ] $ ldd /usr/lib/legacy/glibc/libutil.so.1
        /usr/lib/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007f8ac9932000)
        linux-vdso.so.1 (0x00007ffe239c9000)
kexybiscuit@ProArtB550-CREATOR [ ~ ] $ ldd /usr/lib/libc.so.6
        /usr/lib/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007f2d599cb000)
        linux-vdso.so.1 (0x00007ffedb7ee000)
```

**Pull request merged**